DirectProgramming/C++/ParallelPatterns/openmp_reduction/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 set(CMAKE_CXX_COMPILER "icpx")
-# Set default build type to RelWithDebInfo if not specified
+# Set default build type to RelWithDebInfo if not specified.
```

Comment:
This change is completely unrelated and should go in a separate PR.

If you really insist on combining these, please update the PR description.